### ISSUE TITLE
WCAG 4.1.1: id attribute value must be unique (filter[height="\31 60\%"][width="\31 80\%"][y="-\33 0\%"]:nth-child(1))

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2279]
#### Issue
`id attribute value must be unique` ([duplicate-id](https://dequeuniversity.com/rules/axe/3.2/duplicate-id?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](https://makecode.microbit.org/#editor)
#### Element path
#sidedocsframe;svg[viewBox="\30 \ 0\ 168\.015625\ 56"] > defs > filter[height="\31 60\%"][width="\31 80\%"][y="-\33 0\%"]:nth-child(1)
#### Snippet
`<filter id="blocklyStackGlowFilter1660520634760847" height="160%" width="180%" y="-30%" x="-40%">`
#### How to fix
    Fix any of the following:
      Document has multiple static elements with the same id attribute
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

tons of these errors if you open the side docs to reference; seems to be one per codecard of each of these filters / patterns, all with the same id (should be a class I'm guessing?) 

![Screen Shot 2019-07-17 at 5 51 15 PM](https://user-images.githubusercontent.com/5615930/61421342-8a0d7800-a8bb-11e9-8ad6-69969e5f5466.png)

**Comment by abchatra** [comment]
Fixed
